bots/tests-scan

```diff
@@ -69,7 +69,12 @@ REPO_BRANCH_CONTEXT = {
     },
     'weldr/lorax': {
         'master': [
-            'cockpit/fedora-30',
+            'cockpit/fedora-30/TestSanity',
+            'cockpit/fedora-30/TestImages',
+            'cockpit/fedora-30/TestAWS',
+            'cockpit/fedora-30/TestAzure',
+            'cockpit/fedora-30/TestOpenStack',
+            'cockpit/fedora-30/TestVmware',
```

Comment:
If you move these to `_manual`, we could land these right now. Then you can trigger these manually, but we don't risk making every lorax PR red in case they fail. Once they have shown to work, we can move them to `master`.